### ISSUE TITLE
Cannot find a differ supporting object 'undefined'

### ISSUE THREAD

**Issue by Awatatah** [fetrarij/ng-tempusdominus-bootstrap4#2]
Hello, with below code, where **ngModel** `message.expire_at` already has a timestamp value e.g. `2019-01-20T13:54:00.000Z` but I receive the below error:

I am using `jquery 3.3.1`,  `moment 2.22.2`, `bootstrap4 4.1.1`, `tempusdominus-core 5.0.0`, `tempusdominus-bootstrap-4 5.0.1`, `ngx-tempusdominus-bootstrap 1.0.0`

**html code**
```
<div class="col-5">
    <div class="form-group">
         <div class="input-group date" data-target-input="nearest" NgTempusdominusBootstrap>
           <input
           [(ngModel)]="message.expire_at"
           (ngModelChange)="update()"
           [options]="startOptions"
            NgTempusdominusBootstrapInput
           type="text" class="form-control"
            />
            <div class="input-group-append" NgTempusdominusBootstrapToggle>
                <div class="input-group-text"><i class="fa fa-calendar"></i></div>
            </div>
        </div>
    </div>
</div>
```

**console error**
```
MessageShowComponent.html:8 ERROR Error: Cannot find a differ supporting object 'undefined'
    at KeyValueDiffers.push../node_modules/@angular/core/fesm5/core.js.KeyValueDiffers.find (core.js:6467)
    at NgTempusdominusBootstrapInputDirective.push../node_modules/ngx-tempusdominus-bootstrap/esm5/ngx-tempusdominus-bootstrap.js.NgTempusdominusBootstrapInputDirective.ngOnInit (ngx-tempusdominus-bootstrap.js:106)
    at checkAndUpdateDirectiveInline (core.js:9042)
    at checkAndUpdateNodeInline (core.js:10306)
    at checkAndUpdateNode (core.js:10268)
    at debugCheckAndUpdateNode (core.js:10901)
    at debugCheckDirectivesFn (core.js:10861)
    at Object.eval [as updateDirectives] (MessageShowComponent.html:14)
    at Object.debugUpdateDirectives [as updateDirectives] (core.js:10853)
    at checkAndUpdateView (core.js:10250)

  | View_MessageShowComponent_1 | @ | MessageShowComponent.html:8
-- | -- | -- | --
  | push../node_modules/@angular/core/fesm5/core.js.DebugContext_.logError | @ | core.js:11105
  | push../node_modules/@angular/core/fesm5/core.js.ErrorHandler.handleError | @ | core.js:1670
  | (anonymous) | @ | core.js:4521
  | push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invoke | @ | zone.js:388
  | push../node_modules/zone.js/dist/zone.js.Zone.run | @ | zone.js:138
  | push../node_modules/@angular/core/fesm5/core.js.NgZone.runOutsideAngular | @ | core.js:3716
  | push../node_modules/@angular/core/fesm5/core.js.ApplicationRef.tick | @ | core.js:4521
  | (anonymous) | @ | core.js:4406
  | push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invoke | @ | zone.js:388
  | onInvoke | @ | core.js:3757
  | push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invoke | @ | zone.js:387
  | push../node_modules/zone.js/dist/zone.js.Zone.run | @ | zone.js:138
  | push../node_modules/@angular/core/fesm5/core.js.NgZone.run | @ | core.js:3671
  | next | @ | core.js:4406
  | schedulerFn | @ | core.js:3488
  | push../node_modules/rxjs/_esm5/internal/Subscriber.js.SafeSubscriber.__tryOrUnsub | @ | Subscriber.js:195
  | push../node_modules/rxjs/_esm5/internal/Subscriber.js.SafeSubscriber.next | @ | Subscriber.js:133
  | push../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber._next | @ | Subscriber.js:77
  | push../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber.next | @ | Subscriber.js:54
  | push../node_modules/rxjs/_esm5/internal/Subject.js.Subject.next | @ | Subject.js:47
  | push../node_modules/@angular/core/fesm5/core.js.EventEmitter.emit | @ | core.js:3480
  | checkStable | @ | core.js:3726
  | onLeave | @ | core.js:3793
  | onInvokeTask | @ | core.js:3751
  | push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask | @ | zone.js:420
  | push../node_modules/zone.js/dist/zone.js.Zone.runTask | @ | zone.js:188
  | push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask | @ | zone.js:496
  | invokeTask | @ | zone.js:1540
  | globalZoneAwareCallback | @ | zone.js:1566
  | error (async) |   |  
  | customScheduleGlobal | @ | zone.js:1666
  | push../node_modules/zone.js/dist/zone.js.ZoneDelegate.scheduleTask | @ | zone.js:407
  | onScheduleTask | @ | zone.js:297
  | push../node_modules/zone.js/dist/zone.js.ZoneDelegate.scheduleTask | @ | zone.js:401
  | push../node_modules/zone.js/dist/zone.js.Zone.scheduleTask | @ | zone.js:232
  | push../node_modules/zone.js/dist/zone.js.Zone.scheduleEventTask | @ | zone.js:258
  | (anonymous) | @ | zone.js:1831
  | (anonymous) | @ | http.js:1103
  | push../node_modules/rxjs/_esm5/internal/Observable.js.Observable._trySubscribe | @ | Observable.js:42
  | push../node_modules/rxjs/_esm5/internal/Observable.js.Observable.subscribe | @ | Observable.js:28
  | push../node_modules/rxjs/_esm5/internal/observable/ConnectableObservable.js.ConnectableObservable.connect | @ | ConnectableObservable.js:34
  | push../node_modules/rxjs/_esm5/internal/operators/refCount.js.RefCountOperator.call | @ | refCount.js:19
  | push../node_modules/rxjs/_esm5/internal/Observable.js.Observable.subscribe | @ | Observable.js:23
  | push../node_modules/angular2-token/fesm5/angular2-token.js.Angular2TokenService.handleResponse | @ | angular2-token.js:854
  | push../node_modules/angular2-token/fesm5/angular2-token.js.Angular2TokenService.request | @ | angular2-token.js:825
  | push../node_modules/angular2-token/fesm5/angular2-token.js.Angular2TokenService.get | @ | angular2-token.js:685
  | push../src/app/services/auth.service.ts.AuthService.get2 | @ | auth.service.ts:159
  | push../src/app/admin/messages/message-show.component.ts.MessageShowComponent.getUsers | @ | message-show.component.ts:47
  | push../src/app/admin/messages/message-show.component.ts.MessageShowComponent.ngOnInit | @ | message-show.component.ts:42
  | checkAndUpdateDirectiveInline | @ | core.js:9042
  | checkAndUpdateNodeInline | @ | core.js:10306
  | checkAndUpdateNode | @ | core.js:10268
  | debugCheckAndUpdateNode | @ | core.js:10901
  | debugCheckDirectivesFn | @ | core.js:10861
  | (anonymous) | @ | MessageShowComponent…gfactory.js? [sm]:1
  | debugUpdateDirectives | @ | core.js:10853
  | checkAndUpdateView | @ | core.js:10250
  | callViewAction | @ | core.js:10491
  | execEmbeddedViewsAction | @ | core.js:10454
  | checkAndUpdateView | @ | core.js:10251
  | callViewAction | @ | core.js:10491
  | execComponentViewsAction | @ | core.js:10433
  | checkAndUpdateView | @ | core.js:10256
  | callWithDebugContext | @ | core.js:11143
  | debugCheckAndUpdateView | @ | core.js:10821
  | push../node_modules/@angular/core/fesm5/core.js.ViewRef_.detectChanges | @ | core.js:8637
  | (anonymous) | @ | core.js:4514
  | push../node_modules/@angular/core/fesm5/core.js.ApplicationRef.tick | @ | core.js:4514
  | (anonymous) | @ | core.js:4406
  | push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invoke | @ | zone.js:388
  | onInvoke | @ | core.js:3757
  | push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invoke | @ | zone.js:387
  | push../node_modules/zone.js/dist/zone.js.Zone.run | @ | zone.js:138
  | push../node_modules/@angular/core/fesm5/core.js.NgZone.run | @ | core.js:3671
  | next | @ | core.js:4406
  | schedulerFn | @ | core.js:3488
  | push../node_modules/rxjs/_esm5/internal/Subscriber.js.SafeSubscriber.__tryOrUnsub | @ | Subscriber.js:195
  | push../node_modules/rxjs/_esm5/internal/Subscriber.js.SafeSubscriber.next | @ | Subscriber.js:133
  | push../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber._next | @ | Subscriber.js:77
  | push../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber.next | @ | Subscriber.js:54
  | push../node_modules/rxjs/_esm5/internal/Subject.js.Subject.next | @ | Subject.js:47
  | push../node_modules/@angular/core/fesm5/core.js.EventEmitter.emit | @ | core.js:3480
  | checkStable | @ | core.js:3726
  | onHasTask | @ | core.js:3770
  | push../node_modules/zone.js/dist/zone.js.ZoneDelegate.hasTask | @ | zone.js:441
  | push../node_modules/zone.js/dist/zone.js.ZoneDelegate._updateTaskCount | @ | zone.js:461
  | push../node_modules/zone.js/dist/zone.js.Zone._updateTaskCount | @ | zone.js:285
  | push../node_modules/zone.js/dist/zone.js.Zone.runTask | @ | zone.js:205
  | drainMicroTaskQueue | @ | zone.js:595
  | push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask | @ | zone.js:500
  | invokeTask | @ | zone.js:1540
  | globalZoneAwareCallback | @ | zone.js:1566

```

**Comment by fetrarij** [comment]
hello, how do you define `startOptions` ?

**Comment by Awatatah** [comment]
Ha! My mistake, I removed  `[options]="startOptions"` and the error is now gone, but now I have another issue. The input is not showing the current `message.expire_at` value.

I changed the input to 
```
<input
name="message.expire_at" 
[(ngModel)]="message.expire_at"
NgTempusdominusBootstrapInput
type="text" class="form-control"
 />
```

Screeshot
![screen shot 2018-07-09 at 10 47 37 am](https://user-images.githubusercontent.com/17914534/42464215-93853188-8365-11e8-88b9-69f6a44a9970.png)
